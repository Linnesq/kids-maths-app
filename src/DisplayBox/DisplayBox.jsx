import React from 'react';
import PropTypes from 'prop-types';

import './DisplayBox.scss';

const DisplayBox = (props) => {
    return (
        <div className="DisplayBox">
            <span>{props.value}</span>
        </div>
    );
};

DisplayBox.propTypes = {
    value: PropTypes.number,
};

export default DisplayBox;