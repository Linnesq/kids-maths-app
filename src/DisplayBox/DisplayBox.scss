@import "../variables.scss";

.DisplayBox {
  width: 20%;
  text-align: center;
  font-size: $textRegular;
}
