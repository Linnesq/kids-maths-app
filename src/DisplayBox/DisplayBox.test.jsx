import React from 'react';
import { render } from '@testing-library/react';

import DisplayBox from './DisplayBox';

test('should match snapshot', () => {
    const box = render(<DisplayBox value={9}/>);
    expect(box.container).toMatchSnapshot();
});