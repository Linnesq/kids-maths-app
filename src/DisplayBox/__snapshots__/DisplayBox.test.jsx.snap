// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div>
  <div
    class="DisplayBox"
  >
    <span>
      9
    </span>
  </div>
</div>
`;
